lib/puppet/functions/extlib/commit_date.rb

```diff
@@ -0,0 +1,25 @@
+Puppet::Functions.create_function(:'extlib::commit_date', Puppet::Functions::InternalFunction) do
+  dispatch :commit_date do
+    scope_param
+    param 'String', :path
```

Comment:
`Stdlib::Unixpath` ?

Reply:
Why a unix path? Can't it handle Windows paths?